spec/facility_factory_spec.rb

```diff
@@ -0,0 +1,56 @@
+require 'spec_helper'
+
+RSpec.describe FacilityFactory do
+  describe "Iteration 3" do
+    it "exists" do
+      facility = FacilityFactory.new
+      
+      expect(facility).to be_a(FacilityFactory)
+    end
+
+    it "creates new facilities from oregon data set" do
```

Comment:
These tests are really good. Just be careful expecting them to eq the exact values. Since we are pulling this info from an external source, if that source changes your tests will break. Instead you can test to datatypes. So for example on line 16 you could expect it to `be_a(String)`.